app-games/src/main/java/com/aptoide/android/aptoidegames/feature_payments/wallet/WalletInstallationScreen.kt

```diff
@@ -0,0 +1,388 @@
+      walletApp = walletApp,
+      modifier = Modifier
+        .wrapContentHeight()
+        .padding(all = 16.dp),
+      onWalletInstallStarted = onWalletInstallStarted,
+      onWalletInstallationCanceled = onWalletInstallationCanceled
+    )
+  }
+}
+
+@Composable
+fun WalletInstallationLandscapeView(
+  walletApp: App,
+  buyingPackage: String? = null,
+  onWalletInstallStarted: () -> Unit,
+  onWalletInstallationCanceled: () -> Unit,
+) {
+  val packageText = buyingPackage ?: "Default Package"
+  Row(
+    modifier = Modifier
+      .wrapContentHeight()
+      .fillMaxWidth()
+      .padding(vertical = 24.dp)
+      .verticalScroll(rememberScrollState()),
+  ) {
+    Column(
+      modifier = Modifier
+        .padding(horizontal = 10.dp)
+        .fillMaxWidth(0.4f),
+      horizontalAlignment = Alignment.CenterHorizontally,
+    ) {
+      Image(
+        imageVector = getBonus(Palette.Black, Palette.Primary),
+        contentDescription = null
+      )
+      PurchaseInfoRow(modifier = Modifier.padding(top = 8.dp), buyingPackage = packageText)
+    }
+    PayWithWalletInstallationCard(
+      walletApp = walletApp,
+      modifier = Modifier.padding(end = 24.dp),
+      hasDivider = false,
+      onWalletInstallStarted = onWalletInstallStarted,
+      onWalletInstallationCanceled = onWalletInstallationCanceled
+    )
+  }
+}
+
+@Composable
+fun PayWithWalletInstallationCard(
+  walletApp: App,
+  modifier: Modifier = Modifier,
+  hasDivider: Boolean = true,
+  onWalletInstallStarted: () -> Unit = {},
+  onWalletInstallationCanceled: () -> Unit = {},
+) {
+  if (hasDivider) {
+    Divider(
+      modifier = Modifier.padding(top = 24.dp, bottom = 8.dp, start = 16.dp, end = 16.dp),
+      color = Palette.GreyLight
+    )
+  } else {
+    Spacer(modifier = Modifier.height(16.dp))
+  }
+  Box(
+    modifier = modifier
+      .fillMaxWidth()
+      .background(Palette.White)
+  ) {
+    Column()
+    {
+      Row(
+        verticalAlignment = Alignment.CenterVertically
+      ) {
+        Image(
+          imageVector = getAppcoinsClearLogo(Palette.AppCoinsPink),
+          contentDescription = null,
+        )
+        Text(
+          modifier = Modifier.padding(start = 8.dp),
+          text = stringResource(R.string.iap_appc_title),
+          style = AGTypography.InputsM,
+          maxLines = 2,
+          overflow = TextOverflow.Ellipsis,
+          color = Palette.Black
+        )
+      }
+      Text(
+        modifier = Modifier.padding(),
+        text = stringResource(R.string.iap_appc_body),
+        maxLines = 2,
+        overflow = TextOverflow.Ellipsis,
+        style = AGTypography.Body,
+        color = Palette.Black
+      )
+      AppInstallationViewSmall(
+        app = walletApp,
+        modifier = Modifier.padding(top = 16.dp, bottom = 24.dp),
+        onInstallStarted = onWalletInstallStarted,
+        onCancel = onWalletInstallationCanceled,
+      )
+    }
+  }
+}
+
+@Composable
+fun AppInstallationViewSmall(
```

Comment:
This composable seems to be the same as AppItem. Maybe AppItem can be reused here. Additionally, the install should be cancellable according to design